lib/resources/trending_movies_repository.dart

```diff
@@ -0,0 +1,8 @@
+import '../models/item_model.dart';
+import 'trending_movies_api_provider.dart';
+
+class TrendingMoviesRepository{
+  final trendingMoviesApiProvider = TrendingMoviesApiProvider();
+
+  Future<ItemModel> fetchTrendingMovies() => trendingMoviesApiProvider.fetchTrendingMovies();
+}
```

Comment:
You missed a newline at end of file here.

Reply:
Done!